src/cocoa/toga_cocoa/widgets/canvas.py

```diff
@@ -240,15 +240,7 @@ def reset_transform(self, draw_context, *args, **kwargs):
     # Text
 
     def measure_text(self, text, font, tight=False):
-        textAttributes = NSMutableDictionary.alloc().init()
-        textAttributes[NSFontAttributeName] = font._impl.native
-        text_string = NSAttributedString.alloc().initWithString_attributes_(
-            text,
-            textAttributes
-        )
-        size = text_string.size()
-        size.width += 3
-        return size.width, size.height
+        return font.bind(self.interface.factory).measure(text, tight=False)
```

Comment:
isn't it supposed to be `tight=tight` here? you are ignoring the user input.

Reply:
Good catch! Fixed.